apps/classroomio-com/src/routes/contact/+page.svelte

```diff
@@ -0,0 +1,87 @@
+<script lang="ts">
+  import { slide } from 'svelte/transition';
+  import PageSignupCTA from '$lib/PageSignupCTA/index.svelte';
+  import PageHeader from '$lib/PageHeader/PageHeader.svelte';
+  import Forms from '$lib/ContactUs/Forms/Index.svelte';
+  import { FORM_TYPE } from '$lib/types';
+
+  let currentForm: keyof typeof FORM_TYPE;
```

Comment:
Abstact `keyof typeof FORM_TYPE` to its own type cause I see you are using it more than once

```
type FORM_TYPE_KEY = keyof typeof FORM_TYPE;
```

Reply:
@Chifez